### ISSUE TITLE
Unable to clone repo on Windows

### ISSUE THREAD

**Issue by jmcfarland-swri** [jmcfarland-swri/NASA_ULI_InfoFusion#4]
Using git for Windows, I receive the following error:
```
fatal: cannot create directory at 'src/NATS/utility/node-v8.11.1-linux-x64/lib/node_modules
/npm/node_modules/libnpx/node_modules/yargs/node_modules/cliui/node_modules/string-
width/node_modules/is-fullwidth-code-point/node_modules/number-is-nan': Filename too long
```

It may be possible to use a version of git provided through mingw or cygwin.

**Comment by jmcfarland-swri** [comment]
I did verify that I am able to clone the repo using the version of git that comes with msys/mingw.

**Comment by edecarlo-swri** [comment]
I am getting a similar but different error in my Anaconda window but not in my mingw window when I run `git status.` In my Anaconda window, it says the following for several similar files:

> warning: could not open directory 'src/NATS/utility/node-v8.11.1-linux-x64/lib/node_modules/npm/node_modules/update-notifier/node_modules/latest-version/node_modules/package-json/node_modules/registry-url/node_modules/rc/node_modules/strip-json-comments/': Filename too long

And then it lists them all as modified files that I need to "git add".

**Comment by jmcfarland-swri** [comment]
Yes, I think that is the same issue.  Until it is resolved, you probably need to stick to running git from the msys or mingw prompt.
